common/src/main/java/com/redhat/cloud/notifications/models/CamelProperties.java

```diff
@@ -100,6 +100,22 @@ public void setBasicAuthenticationSourcesId(Long basicAuthenticationSourcesId) {
         this.basicAuthenticationSourcesId = basicAuthenticationSourcesId;
     }
 
+    public Long getBearerAuthenticationSourcesId() {
+        return null;
+    }
+
+    public void setBearerAuthenticationSourcesId(Long bearerAuthenticationSourcesId) {
+        // do nothing here
```

Comment:
Are these placeholders for later in this PR or do plan to create a separate PR for this one?

Reply:
That's it